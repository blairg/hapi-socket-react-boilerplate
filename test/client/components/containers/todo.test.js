/* eslint-disable no-undef */
/* eslint-disable react/jsx-filename-extension */
/* eslint-disable import/extensions */

import React from 'react';
import renderer from 'react-test-renderer';

import Todo from './../../../../src/client/components/containers/todo.jsx';

describe('<Todo />', () => {
  test('should render correctly', () => {
    const entry = { title: 'My Title', body: 'My Body', timestamp: 10000 };

    const tree = renderer.create(<Todo entry={entry} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
