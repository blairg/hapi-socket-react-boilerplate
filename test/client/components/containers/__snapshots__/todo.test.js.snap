// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Todo /> should render correctly 1`] = `
<div
  className="tile"
>
  <h2>
    My Title
  </h2>
  <p
    className="text"
  >
    My Body
  </p>
  <span
    className="text"
  >
    
  </span>
</div>
`;
